chat-plugins/lottery.js

```diff
@@ -0,0 +1,114 @@
+/**
+ *
+ * Lottery.js Made By Dragotic.
+ * A Lottery plugin that allows you to draw lottery and gain great rewards.
+ */
+
+'use strict';
+
+const color = require('../config/color');
+
+// Stores all the lottery tickets
+let allLotteryTickets = [];
+
+Exiled.getAllTickets = function () {
+	if (!Object.keys(Db('tickets').object()).length) return false;
+    // Get all the tickets.
+	for (let i in Db('tickets').object()) {
+	    for (let v in Db('tickets').object()[i]['Tickets']) {
+		    allLotteryTickets.push(Db('tickets').object()[i]['Tickets'][v]);
+	    }
+	}
+};
+
+// Generates Tickets
+function generateTicket() {
+	let chars = 'abcdefghijklmnopqrstuvwxyz0123456789';
+	let lotteryTicket = '';
+	while (lotteryTicket.length < 8) {
```

Comment:
U can use let id = require('uuid/v4'); it's an offline lib so nothing to be added in package.json